tests/python/contrib/test_hexagon/test_vtcm_bandwidth.py

```diff
@@ -113,17 +114,17 @@ class TestMatMulVec:
 
     # Removed most of these to speedup CI.
     size = tvm.testing.parameter(
-        # 10 * KB,
-        # 20 * KB,
-        # 40 * KB,
-        # 80 * KB,
-        # 160 * KB,
-        # 320 * KB,
+        10 * KB,
+        20 * KB,
+        40 * KB,
+        80 * KB,
+        160 * KB,
+        320 * KB,
         640 * KB,
-        # MB,
-        # 2 * MB,
-        # 3 * MB,
-        # 4 * MB,
+        MB,
+        2 * MB,
+        3 * MB,
+        4 * MB,
```

Comment:
Same note here 

ie
```
        if tvm.testing.utils.IS_IN_CI and (size > 10 * KB):
            pytest.skip("Skipping test since it takes too long in CI.")
```

Reply:
Yes indeed.  I will clean all this up before publishing the PR.